adapters/MyTarget/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## myTarget iOS Mediation Adapter Changelog
 
+#### [Version 5.14.2.1](In progress)
```

Comment:
Could you remove the brackets like so,  #### Version 5.14.2.1 (In progress)    
Also, the new myTarget iOS SDK is v5.14.3.   so the new Adapter version 5.14.3.0

Reply:
Ok, no probs.